atlasdb-cassandra/src/test/java/com/palantir/atlasdb/keyvalue/cassandra/CqlExecutorTest.java

```diff
@@ -57,16 +60,23 @@ public void before() {
             Uninterruptibles.sleepUninterruptibly(queryDelayMillis, TimeUnit.MILLISECONDS);
             return result;
         });
+
+        CqlPreparedResult preparedResult = new CqlPreparedResult();
+        preparedResult.setItemId(1);
+        when(queryExecutor.prepare(any(), any())).thenReturn(preparedResult);
+
+        when(queryExecutor.executePrepared(eq(1), any())).thenReturn(result);
     }
 
     @Test
     public void getTimestampsForGivenRows() {
         String expected = "SELECT key, column1, column2 FROM \"foo__bar\""
-                + " WHERE key IN (0x0102,0x0509) LIMIT 100;";
+                + " WHERE key = ? LIMIT 100;";
 
         executor.getTimestamps(TABLE_REF, ImmutableList.of(ROW, END_ROW), LIMIT);
```

Comment:
We should sanity check the return values here too. I believe we have ETE tests for sweep that would ensure that the loading is done correctly. Maybe worth running the large internal product tests once as they are more comprehensive and compare bytes etc.